### DISCUSSION TITLE
I can not clone my local git repo into GitHub

### DISCUSSION THREAD

**Discussion by stevespages** [community/community#21531] [New to GitHub]
<p>I am new to git and GitHub. I have created some git repos on my local computer. Now I want to get what I think is called mirrors of these repos onto GitHub so I can keep them synced with the repos on my local machine. I thought cloning was the way to do this. It looks like it is working during the bash session but when I go to my GitHub Account I can see no sign of the new-project repo. I can see the repo I made from within GitHub. I have pasted the bash session below. Any help would be very much appreciated.</p>
<p>steve@laptop ~ $ ls dir1<br>
project<br>
steve@laptop ~ $ ls dir2<br>
steve@laptop ~ $ git clone dir1/project dir2/new-project<br>
Cloning into ‘dir2/new-project’…<br>
done.<br>
steve@laptop ~ $ ls dir2<br>
new-project<br>
steve@laptop ~ $ git clone dir1/project <a href="https://github.com/stevespages/new-project" rel="nofollow noopener">https://github.com/stevespages/new-project</a><br>
Cloning into ‘<a href="https://github.com/stevespages/new-project" rel="nofollow noopener">https://github.com/stevespages/new-project</a>’…<br>
done.<br>
steve@laptop ~ $</p>

**Comment by Wabri** [comment, marked as answer]
<p>You have to make the remote repository otherwise doesn’t exists this repo <a href="https://github.com/stevespages/new-project" rel="nofollow noopener">https://github.com/stevespages/new-project</a>. you can not clone something to somewhere that doesn’t exists, so you have to create the remote repository in github:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/b/bdb8a852fd78ba2d87f53da280e34165ef0a81c5.png" data-download-href="/uploads/short-url/r4lRRJTncXSTLDrOllwEr4zrS1n.png?dl=1" title="repo.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/12409541/181065336-1ed0d394-63d2-4c72-8ff8-304a86df425e.png" alt="repo.png" data-base62-sha1="r4lRRJTncXSTLDrOllwEr4zrS1n" width="677" height="500" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/b/bdb8a852fd78ba2d87f53da280e34165ef0a81c5_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">repo.png</span><span class="informations">714×527 32.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>and then connect the two repository (your local and your origin) with the command:</p>
<pre><code>git remote add origin https://github.com/stevespages/new-project.git
</code></pre>
<p>when you will make some change you can add-commit-push and see the result!</p>
<p>PS: if you are in your first use of git probably <a href="https://guides.github.com/activities/hello-world/" rel="nofollow noopener">this is necessary</a></p>

**Comment by stevespages** [comment]
<p>Thanks for your reply. I have now managed to do it with your help. I had already done the hello-world activity which you suggested. It was definitely useful too.</p>
